beaker.gemspec

```diff
@@ -53,6 +53,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'stringify-hash', '~> 0.0'
   s.add_runtime_dependency 'beaker-hiera', '~> 0.0'
   s.add_runtime_dependency 'beaker-hostgenerator'
+  s.add_runtime_dependency 'thor'
```

Comment:
The things I'm most familiar with are minimal (`optparse` and its variants); I'm passingly aware of `thor`, `commander`, `GLI`, `trollop` and `slop`, but don't have strong opinions on them.